src/Microsoft.AspNetCore.Authentication/RemoteAuthenticationOptions.cs

```diff
@@ -56,6 +56,11 @@ public string DisplayName
         /// </summary>
         public bool SaveTokensAsClaims { get; set; }
 
+        /// <summary>
+        /// Gets or sets the lifetime of the correlation cookie used to prevent cross-site scripting attacks.
```

Comment:
`RemoteAuthenticationTimeout` is good enough.

Yeah, now's not the time to try re-writing Twitter. I think most of that junk was required as part of the OAuth1 or Twitter spec anyways.  Adding the timeout should be sufficient.